terraform/modules/ebs_backup/variables.tf

```diff
@@ -5,3 +5,13 @@ variable "environment" {
 variable "ebs_backup_error_notification_emails" {
   type = list(string)
 }
+
+variable "system_drive_backup_schedule" {
+  type    = string
+  default = "cron(0 23 ? * FRI *)"
```

Comment:
I've not seen ? in these before, does this just mean irrelevant because it's defined by "FRI"?

Reply:
AWS use slightly different cron expressions to everyone else with 6 fields instead of 5, you can either specify day-of-month (`?`, so not specified here) or day-of-week (`FRI`) https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-cron-expressions.html